### PR TITLE
es: Revision and translation

### PR DESCRIPTION
**Pull request by aaossa**
Observations until line 156.

### CODE REVIEW

**Comment by aaossa** [comment]
#121 Finally here's my PR.

File: README-es.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[ Languages: [English](README.md), [Español](README-es.md), [한국어](README-ko.md), [Português](README-pt.md), [Русский](README-ru.md), [Slovenščina](README-sl.md), [中文](README-zh.md) ]
+[ Idiomas: [English](README.md), [Español](README-es.md), [한국어](README-ko.md), [Português](README-pt.md), [Русский](README-ru.md), [Slovenščina](README-sl.md), [中文](README-zh.md) ]
```
Comment (ceoaliongroo) [inline README-es.md]: @jlevy, translate `Languages` in each language look good idea.  What do you think?

File: README-es.md [hunk 2]
```diff
@@ -19,70 +19,70 @@
 
 ![curl -s 'https://raw.githubusercontent.com/jlevy/the-art-of-command-line/master/README.md' | egrep -o '`\w+`' | tr -d '`' | cowsay -W50](cowsay.png)
 
-La fluidez en el terminal es una destreza a menudo abandonada y considerada arcaica, pero esta mejora su flexibilidad y productividad como ingeniero en formas obvia y sutil. Esta es una selección de notas y consejos al usar el terminal que encontré útiles al trabajar en Linux. Algunos consejos son elementales y algunos bastante específicos, sofisticados u oscuros. Esta página no es larga, pero sí usa y recuerda todos los puntos aquí mostrados, ustedes sabrán un montón.
+La fluidez en el terminal es una destreza a menudo abandonada y considerada arcaica, pero esta mejora su flexibilidad y productividad como ingeniero en forma obvia y sutil. Esta es una selección de notas y consejos al usar el terminal que encontré útiles al trabajar en Linux. Algunos consejos son elementales y algunos bastante específicos, sofisticados u oscuros. Esta página no es larga, pero sí usa y recuerda todos los puntos aquí mostrados, sabrá un montón.
```
Comment (ceoaliongroo) [inline README-es.md]: Please: keep `formas` is more near to the source.

Change `sabrá` => `usted sabrá`

File: README-es.md [hunk 3]
```diff
@@ -19,70 +19,70 @@
...
+pero debido al interés mostrado, parece que vale la pena usar Github, donde existen personas más talentosas que fácilmente pueden sugerir mejoras. Si ve un error o algo que podría ser mejor, por favor, cree un issue o PR! (Por supuesto revise la sección meta de PRs/issues primero.)
 
 
 ## Meta
 
 Alcance:
 
-- Esta guía es tanto para el principiante como para el experimentado. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que puedas buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente ahorra tiempo comparada con otras alternativas.
+- Esta guía es tanto para el principiante como para el experimentado. Los objetivos son *diversidad* (todo importa), *especificidad* (dar ejemplos concretos del caso más común), y *concisión* (evitar cosas que no son esenciales o insignificantes que pueda buscar fácilmente en otro lugar). Cada consejo es esencial en alguna situación o significativamente ahorra tiempo comparada con otras alternativas.
```
Comment (ceoaliongroo) [inline README-es.md]: change: `que pueda buscar fácilmente en otro lugar` => `que pueda buscar en otro lugar fácilmente`

File: README-es.md [hunk 4]
```diff
@@ -19,70 +19,70 @@
...
 - Esta escrito para Linux, con excepción de la sección "[Solo para MacOS X](#macos-x-only)". Muchos de los otros puntos aplican o pueden ser instalados en otros Unices o MacOS (o incluso Cygwin).
-- Se enfoca en Bash interactivo, aunque muchos de los consejos se aplican para otros shells y al Bash scripting por lo general.
-- Esta incluye ambos comandos "estándar" Unix, así como aquellos que requieren la instalación especial de un paquete -- siempre que sea suficientemente importante para ameritar su inclusión.
+- El foco está en Bash interactivo, aunque muchos de los consejos se aplican para otros shells y al Bash scripting en general.
```
Comment (ceoaliongroo) [inline README-es.md]: The existent translation is nearest to the source.

File: README-es.md [hunk 5]
```diff
@@ -113,22 +113,22 @@ Notas:
...
 
-- En ssh, conocer cómo hacer un port tunnel con `-L` o `-D` (y de vez en cuando `-R`) es útil, Ej. para acceder sitio web desde un servidor remoto.
+- En ssh, saber cómo hacer un port tunnel con `-L` o `-D` (y de vez en cuando `-R`) es útil, Ej. para acceder sitio web desde un servidor remoto.
 
-- esto puede ser útil para hacer algunas optimizaciones para su configuración ssh; por ejemplo, Este, `~/.ssh/config` contiene la configuración para evitar desconexiones en ciertos entornos de red, utiliza la compresión (cual es útil con scp sobre conexiones con un ancho de banda pequeña), y la multiplexión de los canales para el mismo servidor con un archivo de control local:
+- Puede ser útil hacer algunas optimizaciones a su configuración ssh; por ejemplo, esto: `~/.ssh/config`, contiene la configuración para evitar desconexiones en ciertos entornos de red, utiliza compresión (cual es útil con scp sobre conexiones con un ancho de banda pequeño), y la multiplexión de canales para el mismo servidor con un archivo de control local:
```
Comment (ceoaliongroo) [inline README-es.md]: change `esto: ~/.ssh/config, ` => `~/.ssh/config`

**Comment by ceoaliongroo** [comment]
@aaossa Great Job. 

I did some observations, please check.

File: README-es.md (hunk 2 above)
Comment (aaossa) [inline README-es.md]: I agree with the suggested changes and also change `sí` => `si`.

File: README-es.md (hunk 3 above)
Comment (aaossa) [inline README-es.md]: Agree.

File: README-es.md (hunk 5 above)
Comment (aaossa) [inline README-es.md]: That's definitelly better, agree.

**Comment by aaossa** [comment]
Thanks @ceoaliongroo . I'll make a commit in a few hours. Tomorrow i'll  check the remaining lines.

**Comment by aaossa** [comment]
Some suggestions and questions:

*206*: `**ctrl-v** **[Tab]** ` =>  `**ctrl-v + [Tab]** `, is a common notation when pressing keys simultaneously.
*185*: `tales como la realización de operaciones u operaciones singulares debajo`... is this ok?
*210*: `para edición de binario ` =>  `para edición binaria`.
*236*: `un resumen del montículo "heap summary"` => `un resumen del montón`.
*236*: `descargados al` => `descargados en`.
*256*: `la estadística histórica en` => `estadística sobre`.

File: README-es.md (hunk 1 above)
Comment (jlevy) [inline README-es.md]: Sure. The only problem is, I have to all of them when we add new languages, so I may not preserve this just out of laziness...

**Comment by jlevy** [comment]
Just a note -- let's keep translations semantically identical. As in don't make changes like
```
**ctrl-v** **[Tab]** => **ctrl-v + [Tab]**
```
above. This will cause divergence/confusion across the languages.

(And in the particular case, it's also incorrect, as you don't press them at once.)

Thanks!

File: README-es.md [hunk 6]
```diff
@@ -158,62 +158,62 @@ Notas:
...
 
-- Para JSON, usa [`jq`](http://stedolan.github.io/jq/).
+- Para JSON, use [`jq`](http://stedolan.github.io/jq/).
 
 - Para archivos Excel o CSV, [csvkit](https://github.com/onyxfish/csvkit) provee `in2csv`, `csvcut`, `csvjoin`, `csvgrep`, etc.
 
 - Para Amazon S3, [`s3cmd`](https://github.com/s3tools/s3cmd) es conveniente y [`s4cmd`](https://github.com/bloomreach/s4cmd) es el mas rápido. Hecho por Amazon [`aws`](https://github.com/aws/aws-cli) es esencial para otras tareas relacionadas al AWS.
 
-- Conocer acerca `sort` y `uniq`, incluyendo opciones de uniq `-u` y `-d` -- ver unas líneas más abajo.
+- Conocer acerca de `sort` y `uniq`, incluyendo las opciones de uniq `-u` y `-d` -- ver one-liners más abajo.
```
Comment (ceoaliongroo) [inline README-es.md]: Change:
 `one-liners` => `[one-liners](https://github.com/jlevy/the-art-of-command-line/blob/master/README-es.md#one-liners)`

File: README-es.md [hunk 7]
```diff
@@ -158,62 +158,62 @@ Notas:
...
+- Sepa que `locale` afecta muchas herramientas de la línea de comandos en forma delicada, incluyendo el orden del ordenamiento (compaginación) y rendimiento. La mayoría de las instalaciones de Linux configuran `LANG` u otras variables de localización para la configuración local como US English. Pero tenga en cuenta que el ordenamiento va a cambiar si cambia `locale`. Y también las rutinas i18n pueden hacer que `sort` u otros comandos se ejecuten *muchas veces* más lentos. En algunas situaciones (tales como la realización de operaciones u operaciones singulares debajo) puede de forma segura ignorar las lentas rutinas i18n por completo y utilizar el sort tradicional basado en byte, usando `export LC_ALL=C`.
```
Comment (ceoaliongroo) [inline README-es.md]: Change: 

'locale' => locale  (Remove comillas ' ')
'sort'=> sort

File: README-es.md [hunk 8]
```diff
@@ -158,62 +158,62 @@ Notas:
...
 
-- Conozca aspectos básicos de `awk` y `sed` para manejo de datos. Por ejemplo, sumar todos lo números en la tercera columna de un archivo de texto: `awk '{ x += $3 } END { print x }'`. Esto es probablemente 3X más rápido y 3X más corto que su equivalente en Python.
+- Conozca aspectos básicos de `awk` y `sed` para manejo de datos. Por ejemplo, sumar todos lo números en la tercera columna de un archivo de texto: `awk '{ x += $3 } END { print x }'`. Esto es probablemente 3 veces más rápido y 3 veces más corto que su equivalente en Python.
```
Comment (ceoaliongroo) [inline README-es.md]: `Conozca aspectos básicos` => `Conozca los aspectos básicos`

File: README-es.md [hunk 9]
```diff
@@ -225,39 +225,39 @@ Notas:
 
 ## Depuración del sistema
 
-- Para depuración web, `curl` y `curl -I` son prácticos, o sus equivalentes `wget`, o el más moderno [`httpie`](https://github.com/jakubroztocil/httpie).
+- Para depuración web, `curl` y `curl -I` son prácticos, o sus equivalentes en `wget`, o el más moderno [`httpie`](https://github.com/jakubroztocil/httpie).
```
Comment (ceoaliongroo) [inline README-es.md]: Change:
`sus equivalentes en` => `como sus equivalentes`

File: README-es.md [hunk 10]
```diff
@@ -225,39 +225,39 @@ Notas:
...
 
-- Para el estado del disco/cpu/red, usar `iostat`, `netstat`, `top` (o el mejor `htop`), y (especialmente) `dstat`. Bueno para recibir una idea rápida de lo que está pasando en tu sistema.
+- Para conocer el estado del disco/cpu/red, usar `iostat`, `netstat`, `top` (o el mejor `htop`), y (especialmente) `dstat`. Bueno para recibir una idea rápida de qué está pasando en un sistema.
```
Comment (ceoaliongroo) [inline README-es.md]: Change: 

`un` => `el`

File: README-es.md [hunk 11]
```diff
@@ -225,39 +225,39 @@ Notas:
...
 
-- Para una resumen en mayor profundidad, usar [`glances`](https://github.com/nicolargo/glances). Este se presenta con varios niveles de estadística en un solo terminal. Muy útil para una verificación rápida de varios subsistemas.
+- Para una visión general en mayor profundidad, usar [`glances`](https://github.com/nicolargo/glances). Este se presenta con varios niveles de estadística en un solo terminal. Muy útil para una verificación rápida de varios subsistemas.
 
-- Para conocer el estado de la memoria, ejecutar y entender la salida de `free` y `vmstat`. En particular, tener en cuenta el valor "cached" es memoria mantenida por el kernel Linux como un archivo de cache, entonces efectivamente cuenta como valor para "free".
+- Para conocer el estado de la memoria, ejecutar y entender la salida de `free` y `vmstat`. En particular, tener en cuenta el valor "en caché" es mantenido en memoria por el kernel Linux como un archivo de cache, por lo que efectivamente cuenta como valor para "free".
```
Comment (ceoaliongroo) [inline README-es.md]: `en caché` => `cached`

File: README-es.md [hunk 12]
```diff
@@ -225,39 +225,39 @@ Notas:
...
+- El sistema de depuración de Java es harina de otro costal, pero un truco simple en las JSM de Oracle y otros consta en que puedes ejecutar `kill -3 <pid>` y una traza completa y un resumen del montículo "heap summary" (incluyendo del detalle de la colección de basura generacional, la cual puede ser altamente informativa) serán descargados al stderr/logs. Las herramientas `jps`, `jstat`, `jstack`, `jmap` del JDK son útiles. [SJK tools](https://github.com/aragozin/jvm-tools) son más avanzadas.
 
 - Usar `mtr` como un mejor traceroute, para identificar los problemas en la red.
 
-- Para mirara porque el disco está lleno, `ncdu` ahorra tiempo sobre los comandos usual como `du -sh *`.
+- Para buscar por qué el disco está lleno, `ncdu` ahorra tiempo sobre los comandos usuales como `du -sh *`.
```
Comment (ceoaliongroo) [inline README-es.md]: `buscar por qué` => `examinar por qué`

File: README-es.md [hunk 13]
```diff
@@ -225,39 +225,39 @@ Notas:
...
 
-- Para encontrar cual socket o proceso está utilizando el ancho de banda, prueba `iftop` o `nethogs`.
```
Comment (ceoaliongroo) [inline README-es.md]: Keep `el`

File: README-es.md [hunk 14]
```diff
@@ -225,39 +225,39 @@ Notas:
...
+- Para encontrar cual socket o proceso está utilizando ancho de banda, pruebe `iftop` o `nethogs`.
 
-- La herramienta `ab` (viene con Apache) es útil para una verificación rápida y sucia del rendimiento del servidor web. Para pruebas de carga más complejas, prueba `siege`.
+- La herramienta `ab` (viene con Apache) es útil para una verificación "rápida y sucia" del rendimiento del servidor web. Para pruebas de carga más complejas, pruebe `siege`.
 
-- Para depuración de redes más serias, `wireshark`, `tshark`, o `ngrep`.
+- Para depuración mas seria de redes, `wireshark`, `tshark`, o `ngrep`.
```
Comment (ceoaliongroo) [inline README-es.md]: `Para depuración` => `Para una depuración`

File: README-es.md [hunk 15]
```diff
@@ -225,39 +225,39 @@ Notas:
...
 
-- Usar `/proc`. Este es extraordinariamente útil algunas veces cuando hay problemas de depuración en vivo. Ejemplos: `/proc/cpuinfo`, `/proc/xxx/cwd`, `/proc/xxx/exe`, `/proc/xxx/fd/`, `/proc/xxx/smaps`.
+- Usar `/proc`. Es extraordinariamente útil algunas veces cuando se depuran problemas en vivo. Ejemplos: `/proc/cpuinfo`, `/proc/xxx/cwd`, `/proc/xxx/exe`, `/proc/xxx/fd/`, `/proc/xxx/smaps`.
 
-- Cuando se depura porque algo salió más en el pasado, `sar` puede ser muy útil. Este muestra la estadística histórica en CPU, memoria, red, etc.
+- Cuando se depura porque algo salió mal en el pasado, `sar` puede ser muy útil. Este muestra la estadística histórica en CPU, memoria, red, etc.
 
-- Para sistemas y análisis de rendimiento de mayor profundidad, ver `stap` ([SystemTap](https://sourceware.org/systemtap/wiki)), [`perf`](http://en.wikipedia.org/wiki/Perf_(Linux)), y [`sysdig`](https://github.com/draios/sysdig).
+- Para analisis de sistemas y de rendimiento mas profundos, vea `stap` ([SystemTap](https://sourceware.org/systemtap/wiki)), [`perf`](http://en.wikipedia.org/wiki/Perf_(Linux)), y [`sysdig`](https://github.com/draios/sysdig).
```
Comment (ceoaliongroo) [inline README-es.md]: - `analisis `=> `análisis`
- `mas` => `más`

File: README-es.md [hunk 16]
```diff
@@ -266,17 +266,17 @@ Notas:
...
+- Es notablemente útil en ocasiones que pueda realizar intersección, unión, y diferencia de conjuntos de archivos de texto vía `sort`/`uniq`. Suponga `a` y `b` como archivos de texto que son únicos. Esto es rápido, y trabaja con archivos de tamaño arbitrario, hasta varios gigabytes. (Sort no está limitado por la memoria, aunque quizás necesite utilizar la opción `-T` si `/tmp` está en una pequeña partición de raíz.) Vea también la nota acerca de `LC_ALL` y las opciones de `sort`, `-u` (dejado de lado para clarificar más abajo).
 ```sh
-      cat a b | sort | uniq > c   # c is a union b
-      cat a b | sort | uniq -d > c   # c is a intersect b
-      cat a b b | sort | uniq -u > c   # c is set difference a - b
+      cat a b | sort | uniq > c   # c es a union b
```
Comment (ceoaliongroo) [inline README-es.md]: `c es a union b` => c es a unido con b

File: README-es.md [hunk 17]
```diff
@@ -343,15 +343,15 @@ Algunos ejemplos de comandos reunidos:
...
 
 - `socat`: socket relay y redireccionador de puerto tcp (similar a `netcat`)
 
 - [`slurm`](https://github.com/mattthias/slurm): visualización del tráfico de red
 
-- `dd`: moviliza data entre archivos y dispositivos
+- `dd`: moviliza data entre archivos o dispositivos
 
-- `file`: identifica el tipo de archivo
+- `file`: identifica el tipo de un archivo
```
Comment (ceoaliongroo) [inline README-es.md]: Remove `un`

File: README-es.md [hunk 18]
```diff
@@ -359,31 +359,31 @@ Algunos ejemplos de comandos reunidos:
...
 
 - `comm`: compara archivos ordenados línea por línea
 
-- `pv`: monitor del progreso de datos, a través, de un tubo
+- `pv`: monitorea el progreso de datos a través de un tubo
 
 - `hd` y `bvi`: descarga o edita archivos binarios
 
-- `strings`: extrae textos de archivos binarios
+- `strings`: extrae texto de archivos binarios
```
Comment (ceoaliongroo) [inline README-es.md]: Change
`de` => `desde`

File: README-es.md [hunk 19]
```diff
@@ -391,49 +391,49 @@ Algunos ejemplos de comandos reunidos:
...
 
-- `dmesg`: arranque y sistema de mensajes de error
+- `dmesg`: mensajes de error del arranque y del sistema 
 
 - `hdparm`: manipulación/rendimiento de discos SATA/ATA
 
 - `lsb_release`: información de la distribución de Linux
 
-- `lsblk`: lista de bloques de dispositivos: un árbol de vista de sus discos y particiones de disco
+- `lsblk`: lista dispositivos de bloque: una vista de arbol de sus discos y particiones de disco
```
Comment (ceoaliongroo) [inline README-es.md]: - `lista dispositivos` => `lista de dispositivos`
- `una vista de arbol` => `una vista tipo árbol`

File: README-es.md [hunk 20]
```diff
@@ -442,28 +442,28 @@ Estos son puntos relevantes *solo* en MacOS.
 
 - Administración de paquetes con `brew` (Homebrew) y/o `port` (MacPorts). Estos pueden ser utilizados para instalar en MacOS muchos de los comandos de arriba.
 
-- Copiar la salida de cualquier comando para una aplicación desktop con `pbcopy` y pegar desde una entrada con `pbpaste`.
+- Copie la salida de cualquier comando en una aplicación de escritorio con `pbcopy` y pegue una entrada desde una con `pbpaste`.
```
Comment (ceoaliongroo) [inline README-es.md]: Remove `desde una`

**Comment by ceoaliongroo** [comment]
@aaossa Pretty good the corrections....Awsome!

> 206: **ctrl-v** **[Tab]** => **ctrl-v + [Tab]**, is a common notation when pressing keys simultaneously.
185: tales como la realización de operaciones u operaciones singulares debajo... is this ok?
210: para edición de binario => para edición binaria.
236: un resumen del montículo "heap summary" => un resumen del montón.
236: descargados al => descargados en.
256: la estadística histórica en => estadística sobre.

Answer to you:
206:  Keep original, i'm with @jlevy 
210: Change it 
236: descargados al => descargados en.  Change it 

about `usted` sounds good.

**I open the issue #226  to verify this apart the point**  may need some discussion, and maybe affect other language 
185: tales como la realización de operaciones u operaciones singulares debajo... is this ok?
236: un resumen del montículo "heap summary" => un resumen del montón.
256: la estadística histórica en => estadística sobre.

File: README-es.md (hunk 1 above)
Comment (ceoaliongroo) [inline README-es.md]: @aaossa Keep Language for now, we see this in a separate Pull Request.

Thanks

File: README-es.md (hunk 7 above)
Comment (aaossa) [inline README-es.md]: In all their appearances in this line?

File: README-es.md (hunk 6 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 8 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 9 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 10 above)
Comment (aaossa) [inline README-es.md]: I tried to keep it just like the original: `what's happening on a system`.

File: README-es.md (hunk 11 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 12 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 13 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 14 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 15 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 16 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 17 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 18 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 19 above)
Comment (aaossa) [inline README-es.md]: Done.

File: README-es.md (hunk 20 above)
Comment (aaossa) [inline README-es.md]: Done.

**Comment by aaossa** [comment]
Thanks again @ceoaliongroo , the next commit will include your suggestions except for 185 and 230, i'll wait your answer on those.

I'll keep an eye on that issue. Maybe when it is decided a PR will fix it.

**Comment by ceoaliongroo** [comment]
@aaossa @jlevy this pull request could be close in favor PR #229, include this revision

**Pull request closed without merging**